Backends/Wayland/mirror.lisp

```diff
@@ -0,0 +1,110 @@
+(in-package #:clim-wayland)
+
+;;; Mirror protocol?
+
+(defclass wayland-mirror ()
+  ((window :initarg :window :reader window))
+  (:default-initargs :window (alx:required-argument :window)))
```

Comment:
it may be worth to also manage in this instance the sheet associated with the mirror, this way the reverse lookup will involve only invoking a reader.